app/javascript/components/widget-controller.jsx

```diff
@@ -1,120 +1,108 @@
 import React from 'react';
-import styled from 'styled-components';
 import { mathMod } from 'ramda';
 import FullPanel from './full-panel';
 import SidePanel from './side-panel';
-import Twitter from './twitter';
 import Numbers from './widgets/numbers';
 import Weather from './widgets/weather';
-import Live from './live-stream';
-import lockedIcon from '../../assets/images/locked.svg';
-import unlockedIcon from '../../assets/images/unlocked.svg';
-import { colors } from '../lib/theme';
 
-const IconWrapper = styled.div`
-  position: absolute;
-  left: 20px;
-  top: 20px;
-`;
-
-const Icon = styled.button`
-  height: 21px;
-  width: 16px;
-  border-style: none;
-  background-color: ${colors.black};
-`;
-
-const iconId = 'lock-icon';
+const TWENTYSECONDS = 20000;
```

Comment:
Love using a constant here for this value, but the ref should describe what the value _does_ rather than what it _is_. E.g., `const SWITCH_INTERVAL = ...` or something